docs/framework/wpf/controls/how-to-style-a-row-in-a-listview-that-implements-a-gridview.md

```diff
@@ -8,7 +8,7 @@ helpviewer_keywords:
 ms.assetid: 2e406ba2-70a0-4e62-841f-0934859de76e
 ---
 # How to: Style a Row in a ListView That Implements a GridView
-This example shows how to style a row in a <xref:System.Windows.Controls.ListView> control that implements a <xref:System.Windows.Controls.GridView><xref:System.Windows.Controls.ListView.View%2A> mode.  
+This example shows how to style a row in a <xref:System.Windows.Controls.ListView> control that implements a <xref:System.Windows.Controls.GridView> <xref:System.Windows.Controls.ListView.View%2A> mode.  
```

Comment:
I think it's usually said that "an interface is implemented" but "a class is inherited".

The above sentence uses implements, but GridView is a class.

I also feel the sentence is missing something between `GridView` and `View`?

Reply:
Yeah it's the wrong word. Really it's just "use a gridview as the view mode" This PR though is really correcting the rendering problem this page has.

It's displaying `... implements a GridViewView` and this fixes it so it at least says `... implements a GridView View`.

The article itself is really about styling a row in a listview that is using a gridview view.

I'll edit the PR to adjust that and approve.

Thanks @wenhx